Documentation/ceph-nfs-crd.md

```diff
@@ -0,0 +1,77 @@
+---
+title: NFS CRD
+weight: 31
+indent: true
+---
+
+# Ceph NFS Gateway CRD
+
+
+## Overview
+Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom resource definition. This will spin up a cluster of [NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) servers that coordinate with one another via shared RADOS objects. The servers will be configured for NFSv4.1+ access, as serving earlier protocols can inhibit responsiveness after a server restart.
+
+## Samples
+
+This configuration adds a cluster of ganesha gateways that store objects in the pool cephfs.a.meta and the namespace **
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephNFS
+metadata:
+  name: my-nfs
+  namespace: rook-ceph
+spec:
+  RADOS:
+    # RADOS pool where NFS client recovery data is stored.
+    # In this example the data pool for the "myfs" filesystem is used.
```

Comment:
No, this is already correct. There is no "my-nfs" filesystem.

That said, it might be good to change this over to use the myfs-metadata pool instead, as the metadata pools tend to be set up with greater resiliency in most deployments.

Reply:
I'm not sure to understand then, where is the definition of the CephFS filesystem then? Thanks!

Reply:
The main example is in:

`cluster/examples/kubernetes/ceph/filesystem.yaml`